staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller_test.go

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
There are other files that test other aspects of the controller's behavior.  It might be better if this file had a name that indicated with aspect it is testing.  (Side-eye here for `controller_test.go`).

Reply:
done, put this test in a dedicated file `max_seats_test.go`